js/models/conversations.js

```diff
@@ -1920,6 +1925,7 @@
         const quitGroup = {
           timestamp: Date.now(),
           groupId: this.id,
+          identifier: id,
```

Comment:
Just be wary that we manually sync the message below so we have to be careful to not sync twice.

Reply:
fixed